File: src/EventsManagement.BusinessLogic/DataTransferObjects/EventDTO.cs
using EventsManagement.BusinessLogic.DataTransferObjects.Interfaces;

namespace EventsManagement.BusinessLogic.DataTransferObjects
{
    internal class EventDTO : IEntityDTO
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public DateTime DateAndTime { get; set; }

        public string Venue { get; set; }

        public string Category { get; set; }

        public int MaxNumberOfParticipants { get; set; }

        public byte[]? Image { get; set; }
    }
}

File: src/EventsManagement.WebAPI.Client/src/Models/Events/EventDTO.ts
export interface EventDTO {
    id: number | undefined,
    name: string,
    description: string,
    dateAndTime: string | undefined,
    venue: string,
    category: string,
    currentNumberOfParticipants: number,
    maxNumberOfParticipants: number,
    image: string | null,
}